lms/services/html_service.py

```diff
@@ -1,16 +1,23 @@
+import re
 from html.parser import HTMLParser
 
 
-def strip_html_tags(html: str) -> str:
-    """Get plain text from a string which may contain HTML tags."""
+class WhiteSpaceHTMLParser(HTMLParser):  # pylint:disable=abstract-method
```

Comment:
I initially thought I would have to substantially change this. I didn't. But I think this version is slightly more pythonic (whatever that means). I kept the refactor.